Assets/Scripts/Eskills/EskillsService.cs

```diff
@@ -55,6 +67,19 @@ public void GetRoomInfo(string session, Action<RoomData> success, Action<Eskills
             _eskillsManager.SetScore(session, status, score, success, error);
         }
 
+        public void GetTicket(string ewt, string ticket_id, Action<TicketData> success, Action<EskillsError> error)
+        {
+            _eskillsManager.GetTicket(ewt, ticket_id, success, error);
+        }
+
+        public void JoinQueue(string ewt, MatchParameters matchParameters)
+        {
+            _eskillsManager.JoinQueue(ewt, matchParameters, session => {
+                Debug.Log(session);
+                GetComponent<OnMatchCreatedReceiver>().OnMatchCreated(session);
+                }, error => {});
```

Comment:
you might want to log the error or give the error feedback back to the developer somehow

Reply:
logging the error